lib/resources/r4/soarian_allergy_intolerance.yaml

```diff
@@ -0,0 +1,83 @@
+name: AllergyIntolerance
+field_name_base_url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance
```

Comment:
Can you change the url to https://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance

Reply:
Done